src/lib/Libwin/accesinfo.c

```diff
@@ -496,8 +518,7 @@ secure_file2(char *path, char *user, ACCESS_MASK mask, char *user2, ACCESS_MASK
 	}
 	if (gsid) {
 		if (SetSecurityDescriptorGroup(&sd, gsid, FALSE) == 0) {
-			sprintf(logb, "error setting group for file %s", path);
-			log_err(-1, "secure_file2", logb);
+			log_err(-1, __func__, "failed in SetSecurityDescriptorGroup for %s", path);
```

Comment:
why not print "gname" ?

Reply:
Because at this time it will be inaccessible.